samples/sample/pom.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.microsoft.jfr</groupId>
+    <artifactId>jfr-streaming-samples</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>sample</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <name>sample</name>
+  <description>A sample for Java Flight Recorder Streaming Library using Fibonacci number generator</description>
+  <url>https://github.com/Microsoft/jfr-streaming.git</url>
+
```

Comment:
this does not depend on jfr-streaming so it is not available on the classpath

Reply:
Is this about this `<url>` tag? I added this tag as this sample app is gonna be part of the jfr-streaming repo on git. But this app does depend on the jfr-streaming library. It doesn't use the local code from `src`.

Reply:
no this is just about the pom file in general, it needs to contain:

```
    <dependency>
            <groupId>com.microsoft.jfr</groupId>
            <artifactId>jfr-streaming</artifactId>
    </dependency>
```

Reply:
`samples/pom.xml` has this dependency already.